api/controllers/content.js

```diff
@@ -1,7 +1,26 @@
 'use strict'
 
-module.exports = {
-  get: function (request, response) {
-    response.json([ 'unknown' ])
-  }
+const sequelize = require('../../config/sequelize')
+const {forEach} = require('lodash')
+
+const get = (request, response) => {
+  var uri = removeQueryParameters(request.query['uri'].toLowerCase())
+  sequelize.query(
+    'SELECT events.uri FROM events LEFT JOIN scores USING (uri) WHERE scores.uri IS NULL AND events.uri LIKE(\'' + uri + '%\')',
```

Comment:
Since you're using SQL, I believe you need to escape uri before concatenating it into the string to prevent SQL injection.
See http://docs.sequelizejs.com/manual/tutorial/raw-queries.html#replacements